roles/cluster_upgrade/tasks/upgrade.yml

```diff
@@ -84,6 +100,15 @@
     reboot_after_upgrade: "{{ reboot_after_upgrade }}"
     timeout: "{{ upgrade_timeout }}"
 
+- name: Start ovirt-ha-agent service
+  service:
+    name: ovirt-ha-agent
```

Comment:
Shouldn't we also start the `ovirt-ha-broker`?

Reply:
No because ha-agent will start it. because ha-agent needs ha-broker

Reply:
i verified it locally.